app/views/permissions/show.html.erb

```diff
@@ -11,7 +11,7 @@
         </tr>
       </thead>
       <tbody>
-        <% if @permission.fetch('group_permissions', []).blank? %>
+        <% if @permission.fetch('group_permissions', []).blank? || @permission.fetch('group_permissions', []).all? { |group| group.fetch('permissions', []).blank? } %>
```

Comment:
That is already handled, `show` calls `hydrate_groups` which, if the group is valid calls `hydrate_permissions` which deletes any permissions that are not **read** or **order**